implementations/python/sugar-source-tree/tests/test_for_unroll.py

```
@@ -113,7 +177,7 @@ def test_arity_mismatch_stays_loud():
     test_starred_target_stays_loud()
     test_arity_mismatch_stays_loud()
     test_jump_bearing_body_never_unrolls()
-    test_for_else_splices_after_the_unroll()
+    test_for_else_stays_loud_in_the_for_lane()
```

Ruff (0.15.21)

[error] Undefined name `test_for_else_stays_loud_in_the_for_lane`

(F821)